ts/util/expiringMessages.ts

```diff
@@ -20,8 +49,17 @@ export async function destroyMessagesAndUpdateRedux(
   const conversationWithChanges = uniq(messages.map(m => m.conversationKey));
 
   try {
+    const messageIds = messages.map(m => m.messageId);
+
+    // Delete any attachments
+    // tslint:disable-next-line: prefer-for-of
+    for (let i = 0; i < messageIds.length; i++) {
+      const message = await Data.getMessageById(messageIds[i]);
+      await message?.cleanup();
```

Comment:
no we can't. I am not too sure what would be a good fix here. We should probably send the messageIds to cleanup to the node side and do a single remove there, but depending on how slow that is, it will block the UI thread.

Reply:
or include it in the `removeMessagesByIds` just below directly. But yeah, we don't have MessageModels and stuff like that on the node side, just a json dump. And as I said, it might cause issues with blocking the UI. Let's keep it the way you made it for now.